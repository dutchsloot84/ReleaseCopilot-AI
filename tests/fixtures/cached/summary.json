{
  "total_stories": 3,
  "total_commits": 2,
  "stories_with_commits": 2,
  "stories_without_commits": 1,
  "orphan_commits": 1,
  "generated_at": "2025-10-15T17:23:09-07:00",
  "timezone": "America/Phoenix"
}
