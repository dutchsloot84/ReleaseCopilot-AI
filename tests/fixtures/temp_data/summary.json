{
  "total_stories": 2,
  "total_commits": 2,
  "stories_with_commits": 1,
  "stories_without_commits": 1,
  "orphan_commits": 1
}
