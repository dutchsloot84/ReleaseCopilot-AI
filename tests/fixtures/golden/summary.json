{
  "total_stories": 2,
  "total_commits": 2,
  "stories_with_commits": 1,
  "stories_without_commits": 1,
  "orphan_commits": 1,
  "generated_at": "2025-10-15T17:23:09-07:00",
  "timezone": "America/Phoenix"
}
